frontend/src/components/ParticipantSearch.js

```diff
@@ -35,7 +34,7 @@ const ParticipantSearch = observer(() => {
       <Grid
         container
         component="form"
-        onSubmit={handleSubmit}
+        onSubmit={e => (handleSubmit(e), props.handleToggle())}
```

Comment:
This seems odd to use comma syntax like this.  `handleToggle` should just be part of the handleSubmit code